dfget/config/config.go

```diff
@@ -165,6 +165,9 @@ type Config struct {
 	// default:`p2p`.
 	Pattern string `json:"pattern,omitempty"`
 
+	// CA certificate to verify when supernode download from source
```

Comment:
Not only download from source.  Maybe `Interact with the source` will be better. WDYT?

And so do with the comment of `Insecure`